aws/resource_aws_db_instance.go

```diff
@@ -273,6 +278,41 @@ func resourceAwsDbInstance() *schema.Resource {
 				},
 			},
 
+			"restore_to_point_in_time": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				ForceNew: true,
+				ConflictsWith: []string{
+					"s3_import",
+					"snapshot_identifier",
+					"replicate_source_db",
+				},
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"restore_time": {
+							Type:          schema.TypeString,
+							Optional:      true,
+							ValidateFunc:  validateUTCTimestamp,
+							ConflictsWith: []string{"restore_to_point_in_time.0.use_latest_restorable_time"},
+						},
+						"source_db_instance_identifier": {
```

Comment:
Nit: I like lines between.
```suggestion

						"source_db_instance_identifier": {
```